modules/liquidity-pools/base/src/lib.rs

```diff
@@ -271,4 +275,78 @@ impl<T: Trait<I>, I: Instance> Module<T, I> {
 
 		Ok(())
 	}
+
+	fn _set_identity(who: &T::AccountId, pool_id: LiquidityPoolId, identity_req: &IdentityRequest) -> DispatchResult {
```

Comment:
```suggestion
	fn _set_identity(who: &T::AccountId, pool_id: LiquidityPoolId, identity_req: IdentityRequest) -> DispatchResult {
```

should be possible to avoid clone all the fields

Reply:
Ok. Thought it was an off-chain asset.